### DISCUSSION TITLE
Conversion of Swagger 2.0 to OpenAPI

### DISCUSSION THREAD

**Discussion by 6savage9** [OAI/OpenAPI-Specification#3345] [Q&A] [answered]
I am working on Spring boot and have to convert entire project along with all UI elements to OpenAPI. Currently we are using Swagger 2.0 for the same. I have SwaggerController where we are setting up the documentation details and SwaggerConfigController where we are loading config details. I am not able to understand it in depth and that's why cannot work on its conversion as well. Can someone please help me with this issue

**Comment by handrews** [comment, marked as answer]
There is a tool for this - someone wrote a blog post on it here: https://dev.to/derberg/convert-swagger-2-0-to-openapi-3-0-3joj

(this does not imply the endorsement of the blog post or tool, it's just informational)

**Comment by handrews** [comment]
I'm going to close this as resolved due to lack of further questions.

**Reply by shoradrustum** [reply]
Ok
